nav2_bringup/launch/nav2_default_view.rviz

```diff
@@ -76,20 +76,23 @@ Visualization Manager:
           Value: true
         base_scan:
           Value: false
-        map:
+        caster_back_link:
```

Comment:
I'd leave these turtlebot 3 specific transforms out by default. That makes the config a bit less turtlebot 3 specific, and IMHO they kind of clutter up the display anyhow. I frequently find myself turning them off so I can better see what's going on.

Reply:
I'll only leave the base_link, map and odom.